bundle/src/main/java/com/adobe/acs/commons/util/ResourceDataUtil.java

```diff
@@ -21,49 +21,22 @@
 
 import java.io.IOException;
 import java.io.InputStream;
-import java.io.StringWriter;
 
 import javax.jcr.Node;
 import javax.jcr.RepositoryException;
-import javax.servlet.RequestDispatcher;
 
 import org.apache.commons.io.IOUtils;
-import org.apache.commons.lang.StringUtils;
-import org.apache.sling.api.SlingHttpServletRequest;
-import org.apache.sling.api.SlingHttpServletResponse;
 import org.apache.sling.api.resource.Resource;
 import org.apache.sling.api.resource.ResourceResolver;
 import org.osgi.annotation.versioning.ProviderType;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-import com.adobe.acs.commons.util.BufferedServletOutput.ResponseWriteMethod;
 import com.day.cq.commons.jcr.JcrConstants;
 
 @ProviderType
 @SuppressWarnings({"checkstyle:abbreviationaswordinname", "squid:S1118"})
-public class ResourceDataUtil {
-    public static final String ENCODING_UTF_8 = "UTF-8";
-    private static final Logger log = LoggerFactory.getLogger(ResourceDataUtil.class);
-
-    public static String getIncludeAsString(final String path, final SlingHttpServletRequest slingRequest,
```

Comment:
@reggie7 we cannot remove methods like this; this is an exported API -- removing it would break all code that might depend on it.

Reply:
Oh, sorry, I was thinking locally (there was just the one usage), forgot that someone might use it as a library method simply :) Restored it. Thanks for noticing it!